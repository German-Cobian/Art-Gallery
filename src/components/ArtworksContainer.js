import React from 'react';
import ArtworksHeader from './ArtworksHeader';
import Artworks from './Artworks';

const ArtworksContainer = () => (
  <div className="d-flex flex-column my-5">
    <div className="border-bottom">
      <ArtworksHeader />
    </div>
    <div className="border-bottom">
      <h2>Artworks Container Component</h2>
    </div>
    <div className="border-bottom">
      <Artworks />
    </div>
  </div>
);

export default ArtworksContainer;
