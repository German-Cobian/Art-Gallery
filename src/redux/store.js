import { createStore, combineReducers, applyMiddleware } from 'redux';
import logger from 'redux-logger';
import thunk from 'redux-thunk';
import artworksReducer from './artworksApi';

const reducers = combineReducers({
  artworksReducer,
});

const store = createStore(
  reducers,
  applyMiddleware(logger, thunk),
);

export default store;
