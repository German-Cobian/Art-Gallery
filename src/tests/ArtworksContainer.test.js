import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import '@testing-library/jest-dom/extend-expect';
import ArtworksContainer from '../components/ArtworksContainer';
import store from '../redux/store';

describe('ArtworksContainer component', () => {
  test('matches ArtworksContainer component snapshot', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <ArtworksContainer />
      </Provider>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
