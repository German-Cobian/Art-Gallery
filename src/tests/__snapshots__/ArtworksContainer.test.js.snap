// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArtworksContainer component matches ArtworksContainer component snapshot 1`] = `
<div
  className="artworks-container"
>
  <div
    className="d-flex flex-row"
  >
    <div
      className="my-5 col-8"
    >
      <img
        alt="Guernica by Picasso"
        className="col-10"
        src="Guernica.jpeg"
      />
    </div>
    <div
      className="mt-5 col-6"
    >
      <h2
        className="text-start"
      >
        Art Institute of Chicago
      </h2>
      <p
        className="text-start"
      >
        60 awesome artworks
      </p>
    </div>
  </div>
  <h6
    className="text-start my-3 mx-5"
  >
    DETAILS BY ARTWORK
  </h6>
  <div
    className="d-flex flex-wrap"
  />
</div>
`;
